pkg/component/rsyslogrelpconfigcleaner/rsyslog_relp_config_cleaner.go

```diff
@@ -93,94 +88,9 @@ func (r *rsyslogRelpConfigCleaner) WaitCleanup(ctx context.Context) error {
 
 func (r *rsyslogRelpConfigCleaner) computeResourcesData() (map[string][]byte, error) {
 	var (
-		objects            []client.Object
-		serviceAccountName = "default"
+		objects []client.Object
```

Comment:
You can remove this variable and just pass the `daemonSet` directly here: https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/6dc16d1721fbd2fb9ff260aaebf35e5b868811f8/pkg/component/rsyslogrelpconfigcleaner/rsyslog_relp_config_cleaner.go#L168